src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    public static final int BALLS_NUMBER = 3;
+    public static final int MAX_NUMBER_OF_BALL = 100;
```

Comment:
```suggestion
    private static final int BALLS_NUMBER = 3;
    private static final int MAX_NUMBER_OF_BALL = 100;
```
be attentive, in the previous comments I added this